### PR TITLE
WALL-25: Build Footer Section

### PR DESCRIPTION
**Pull request by manikangkan**
## Summary

- Enhance CustomLink comp
- Global clean ups
- Build Footer Section

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (code change that neither fixes a bug nor adds a feature)

### Linked Issues

- _Reference any related issues here by linking to JIRA or GitHub Issues._

## JIRA Issue(s)

- [JIRA Issue Link](https://wallaceproject.atlassian.net/browse/WALL-25)

## Checklist (check all)

- [x] Automated tests have been added, or an explanation for why not
- [ ] Security implications and dependencies on linked features have been considered and explained

## Screenshots & Evidence
https://66e26f29d4211b0a265b8bc6-qwptqnquhw.chromatic.com/?path=/story/components-sections-footer--default
image
image
image

### CODE REVIEW

File: src/components/Sections/Footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+}) => (
+  <div className="flex flex-col md:flex-row xs:flex-col md:col-span-2 xs:col-span-1">
+    <div className="flex-1">
+      <Heading variant="h4" size="sm" className="text-green-400 mb-[24px] sm:mb-[16px]">
+        {companyInfo.title}
+      </Heading>
+      {companyInfo.locations.map((location, index) => (
+        <div key={index} className="mb-4">
+          <Body size="xsm" className="mb-[12px] text-white/70">
+            {location.name}
```
Comment (raj-edition) [inline src/components/Sections/Footer/Footer.tsx:80]: move subcomponents to sub folder `Sections/Footer/components`

File: src/components/Sections/Footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+        </div>
+      ))}
+    </div>
+    <div className="mt-[32px] space-y-[24px] flex-1">
+      <Body size="md" className="text-electric-green-300">
+        {companyInfo.phone}
+      </Body>
+      <div className="space-x-[24px]">
+        {companyInfo?.socialLinks?.map((link, index) => (
+          <a href={link.href} key={index}>
```
Comment (raj-edition) [inline src/components/Sections/Footer/Footer.tsx:96]: its advisible to avoid index for key

**Review by raj-edition — COMMENTED** [review]
some changes

**Comment by raj-edition** [comment]
image
this variant is missing

**Comment by manikangkan** [comment]
> image this variant is missing

image
image

I'm building this both in a single comp, let the footer go through Raj, I'll update soon

File: src/components/Sections/Footer/Footer.tsx (hunk 1 above)
Comment (manikangkan) [reply src/components/Sections/Footer/Footer.tsx]: sorted

File: src/components/Sections/Footer/Footer.tsx (hunk 2 above)
Comment (manikangkan) [reply src/components/Sections/Footer/Footer.tsx]: sorted

**Review by raj-edition — APPROVED** [review]

**Pull request merged**